api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2Spec.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"replicas", "config", "image",
+        "livenessProbe", "readinessProbe", "jvmOptions",
+        "affinity", "tolerations", "logging", "metrics", "tracing", "template", "clusters", "mirrors"})
+@EqualsAndHashCode(doNotUseGetters = true)
```

Comment:
I think you have to use `@EqualsAndHashCode(callSuper = true, doNotUseGetters = true)`

Reply:
Thanks - I had no idea how to get past the spotbugs error :)